src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -331,7 +331,19 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 			elements: obj.stripeElements,
 			redirect: 'if_required',
 			confirmParams: {
-				return_url: order.redirect_url
+				return_url: order.redirect_url,
+				shipping: {
+					name: obj.getPurchaserData().name,
+					phone: obj.getPurchaserData().phone,
+					address: {
+						line1: $('#tec-tc-purchaser-address1').val(),
+						line2: $('#tec-tc-purchaser-address2').val(),
+						city: $('#tec-tc-purchaser-city').val(),
+						state: $('tec-tc-purchaser-state').val(),
```

Comment:
is this a class or ID? Assuming ID:
```suggestion
						state: $('#tec-tc-purchaser-state').val(),
```